agreement.md

```diff
@@ -0,0 +1,151 @@
+# Aragon Network Agreement
+
+## Section 0. Background
+The Aragon Network is a Decentralized Autonomous Organization that builds and maintains a dispute resolution protocol and other smart contract infrastructure.
+
+The Aragon Network is governed by Aragon Network Token (ANT) holders, who supported the initial creation of the network and under this Agreement will continue to guide its evolution through established governance processes.
+
+This Agreement exists to enumerate the shared mission, principles, and governance guidelines that bind ANT holders in their collaborative effort to create value for the Aragon Network.
+
+## Section 1. RFC 2119 compliance
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+## Section 2. Immutable guidelines
+The guidelines in this section SHALL NOT be amended. If any other guidelines in this Agreement conflict with these immutable guidelines, the immutable guidelines SHALL take priority for enforcement purposes. These guidelines apply to all proposals submitted to the Aragon Network organization.
```

Comment:
Two sets of guidelines are referenced here. To avoid confusion, I'd recommend not using "**These** guidelines..." Better to always distinguish explicitly between "Guidelines in this Agreement" (i.e. all guidelines) and "Guidelines in Section 2" or "Immutable guidelines"